src/OpenFeature/EventExecutor.cs

```diff
@@ -303,6 +313,14 @@ private void ProcessDefaultProviderHandlers(Event e)
         }
     }
 
+    private static string GetClientName(string client)
+    {
+        if (string.IsNullOrEmpty(client))
```

Comment:
Should we check for whitespace?


```suggestion
        if (string.IsNullOrWhiteSpace(client))
```